src/modules/NtpClient.rb

```diff
@@ -281,15 +295,15 @@ def ProcessNtpConf
     # synchronize_time and sync_interval variables
     # Return updated value of synchronize_time
     def ReadSynchronization
-      crontab = SCR.Read(path(".cron"), CRON_FILE, "")
-      log.info("NTP Synchronization crontab entry: #{crontab}")
-      cron_entry = (crontab || []).fetch(0, {}).fetch("events", []).fetch(0, {})
-      @synchronize_time = cron_entry["active"] == "1"
+      return false unless File.exist?(TIMER_PATH)
 
-      sync_interval_entry = cron_entry.fetch("minute", "*/#{DEFAULT_SYNC_INTERVAL}")
-      log.info("MINUTE #{sync_interval_entry}")
+      timer_content = File.read(TIMER_PATH)
+      log.info("NTP Synchronization timer entry: #{timer_content}")
+      @synchronize_time = SCR.Execute(path(".target.bash"),
+        "/bin/systemctl is-active #{TIMER_FILE}").zero?
 
-      @sync_interval = sync_interval_entry.tr("^[0-9]", "").to_i
+      interval = timer_content[/^\s*OnUnitActiveSec=(\d+)m/, 1]
```

Comment:
I do not know the exact syntax, are the spaces around `=` allowed? Maybe the regexp should be less strict about the expected format.